parts/linux/cloud-init/artifacts/cse_helpers.sh

```diff
@@ -384,40 +384,80 @@ should_skip_nvidia_drivers() {
     echo "$should_skip"
 }
 
-start_watch () {
-  capture_time=$(date +%s)
-  start_timestamp=$(date +%H:%M:%S)
+installJq () {
+  # jq is not available until downloaded in install-dependencies.sh with the installDeps function
+  # but it is needed earlier to call the capture_benchmarks function in pre-install-dependencies.sh
+  output=$(jq --version)
+  if [ -n "$output" ]; then
+    echo "$output"
+  else
+    if [[ $OS == $MARINER_OS_NAME ]]; then
+      sudo tdnf install -y jq
```

Comment:
should we capture the error in case it fails to install?
For example, in install_dependencies.sh it is doing something like this `apt_get_install 30 1 600 grub-efi || exit 1`